readme.txt

```diff
@@ -0,0 +1,37 @@
+POINTS TO NOTE:
+
+* bootloader.ld
+
+-> FLASH(rx) : ORIGIN = 0x10000000, LENGTH = 12k //maintain the length as multiple of 4k
+-> Make similar changes in the other .ld files and CMakeLists.txt file
+    combined.ld
+    standalone.ld
+
+Refer the branch - https://github.com/usedbytes/rp2040-serial-bootloader/tree/bl32k and the changes.
```

Comment:
is this correct ?

Reply:
Yes. If we need to use a bootloader which occupies more space i.e > 12k assigned for it during compile time, then the implementation in that branch can be used/referred and changes implemented to suit the needs. 

This issue arises when the Bootloader C code is compiled without optimization. Cmake has the optimisation set to "Debug" which has less optimization. Ideally should use the "Release" which will make the size of the code < or equal to 12k